docs/integrations/sources/google-ads.md

```diff
@@ -82,10 +97,11 @@ The Google Ads source connector supports the following [sync modes](https://docs
 - [Incremental Sync - Append](https://docs.airbyte.com/understanding-airbyte/connections/incremental-append)
 - [Incremental Sync - Deduped History](https://docs.airbyte.com/understanding-airbyte/connections/incremental-deduped-history)
 
-**Important note**:
+:::caution
+When using the Conversion Window feature, please be aware that it may result in duplicates during Incremental Sync. Due to the possibility of conversions happening within the specified window, the connector will extend the range of the sync, which could potentially result in duplicated data.
```

Comment:
This section is a bit vague to the user; "duplicates" isn't exactly right (and makes it sound like we're doing something wrong). This is a feature; it's more that we'd have updated  conversion data for the same campaigns/ad_groups/ad_group_ads. 

A conversion window is; 


> In digital advertising, a "conversion" generally refers to when a user who views or interacts with an ad goes on to perform a desired action. This could be anything from clicking through to the advertiser's website, signing up for a newsletter, making a purchase, and so on.
> Now, the "conversion window" is the period of time after a user sees or clicks on an ad during which their actions can still be credited to that ad.
> For example, imagine an online shoe store runs an ad and sets a conversion window of 30 days. If you click on that ad today, any purchases you make on the shoe store's site within the next 30 days will be considered conversions resulting from that ad.
> The length of the conversion window can vary depending on the goals of the advertiser and the nature of the product or service. Some businesses might set a shorter conversion window if they're promoting a limited-time offer, while others might set a longer window if they're advertising a product that consumers typically take a while to think about before buying.
> In essence, the conversion window is a tool for measuring the effectiveness of an advertising campaign. By tracking the actions users take after viewing or interacting with an ad, businesses can gain insight into how well their ads are working and adjust their strategies accordingly.


I think we can boil all this down to: every time the connector runs it will output conversion data for every campaign, ad_group, or ad_group_ad which was running less than N days ago where N is the configured conversion window. This will make it look like the same campaigns/ad_groups/ads have different conversion numbers when in reality each record tracks the number of conversions for that resource at the time of extracting the data from the Google Ads API. If you want to know the most up-to-date conversion numbers, use incremental-dedupe syncs. 

Maybe we should also have that blurb to explain conversion windows in the doc

Reply:
Gotcha, thanks for the clarification! I added a dedicated section to this with the blurb and explanation, and removed the deduped recommendation